helm/values.yaml

```diff
@@ -95,6 +95,22 @@ controller:
   extraEnv: {}
   envFrom: []
 
+  serviceAccount:
+    create: true
+    name: superblocks-controller
```

Comment:
nit: i think this is returned by a helper

Reply:
can't use helpers in the values file right?